### PR TITLE
Adding interactive prompt for cmd line args

### PR DESCRIPTION
**Pull request by RBlafford**

### CODE REVIEW

**Comment by RBlafford** [comment]
I think this code is kind of fugly. Fuck IO

**Comment by senior7515** [comment]
LGTM. Would be great to have some tests around this JSON parsing. Seems like the kind of test that could benefit from a set of static json parsing tests.

File: concord/runway.py [hunk 1]
```diff
@@ -121,6 +121,47 @@ def present_and_select(message, table_metadata, raw_metadata):
         sys.exit(1)
     return user_selection
 
+def simple_prompt(obj):
+    default_str = "None" if 'default' not in obj else obj['default']
+    return raw_input("Argument:{} (default: {}) ".format(obj['flag_name'], default_str))
```
Comment (senior7515) [inline concord/runway.py]: flag_name is guaranteed here right?

Comment (RBlafford) [inline concord/runway.py]: Yes

**Pull request merged**